openwisp_users/tests/test_models.py

```diff
@@ -53,6 +55,7 @@ def test_organizations_pk(self):
         self.assertIn((org1.pk,), user.organizations_pk)
         self.assertEqual(len(user.organizations_pk), 2)
 
+    @capture_stderr()
```

Comment:
I would not hide this so that we'll remember to remove these obsolete methods in the next version

Reply:
Okay! 👍🏻